lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/config/util/RncRestConfConfigUtils.java

```diff
@@ -106,8 +106,8 @@ public static SecurityConfig createSecurityConfig(RncRestConfConfiguration confi
             keyStore.load(ksFile.get(), config.getKeyStorePassword().toCharArray());
             trustKeyStore.load(trustKsFile.get(), config.getTrustKeyStorePassword().toCharArray());
 
-            return new SecurityConfig(keyStore, config.getKeyStorePassword(), 
-                                    trustKeyStore, config.getTrustKeyStorePassword(), 
+            return new SecurityConfig(keyStore, config.getKeyStorePassword(),
+                                    trustKeyStore, config.getTrustKeyStorePassword(),
```

Comment:
Incorrect formatting.

You have two options:
1. trustKeyStore 't' should be exactly below keyStore 'k'
2. trustKeyStore 't' should be 8 chars after return 'r'

Reply:
I can handle it.